src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelGene.tsx

```diff
@@ -63,7 +65,23 @@ const TrackPanelGene = (props: TrackPanelGeneProps) => {
 
   const { gene } = data;
   const geneTrackStatus = trackStatusGetter(GENE_TRACK_ID);
-  const transcriptTrackStatus = trackStatusGetter(TRANSCRIPT_TRACK_ID);
+
+  const getTranscriptTracks = () =>
+    prepareTranscriptsTrackData(gene).map((transcriptTrackData) => {
+      const transcriptTrackStatus = trackStatusGetter(
+        transcriptTrackData.track_id
+      );
+
+      return (
+        <TrackPanelListItem
+          key={transcriptTrackData.label}
+          categoryName="main"
+          trackStatus={transcriptTrackStatus}
+          defaultTrackStatus={Status.SELECTED}
+          track={transcriptTrackData}
+        />
+      );
+    });
```

Comment:
Remember to only show the canonical transcript in production.

Reply:
✅